### ISSUE TITLE
Investigate PAN001 tracking failure

### ISSUE THREAD

**Issue by wtgee** [panoptes/POCS#351]
Tried to correct Dec for over 30 seconds even though total correction should have be 9ms. Maybe too little of a correction? But the mount should have reported `Tracking` in status for a skip.

This was on the 43rd image and had been working fine until then. Target still about 45 minutes from reaching meridian.

```
D0115 10:44:06.673 observatory.py:277   Offset Info: OffsetError(delta_ra=<Angle 12.562124399983077 arcsec>, delta_dec=<Angle 0.04534232400104088 arcsec>, magnitude=<Angle 12.438572528277144 arcsec>)
D0115 10:44:06.674 machine.py:258       After calling analyze. Now in analyzing state
D0115 10:44:06.676 machine.py:300       Source: analyzing        Dest: tracking
D0115 10:44:06.676 machine.py:161       Transition method: adjust_tracking
D0115 10:44:06.676 machine.py:200       Checking safety for adjust_tracking
D0115 10:44:06.775 core.py:328          Dark Check: True
D0115 10:44:06.777 core.py:364          Weather Safety: True [11 sec old - 2018-01-15 10:43:55.597000]
D0115 10:44:10.937 machine.py:244       Before calling adjust_tracking from analyzing state
I0115 10:44:18.302 messaging.py:122     PANCHAT Checking our tracking
D0115 10:44:18.303 observatory.py:304   Updating the tracking
I0115 10:44:18.304 observatory.py:331   Adjusting Dec: north 9.04 ms 0.05 arcsec
D0115 10:44:29.979 observatory.py:342   Waiting for Dec tracking adjustment
D0115 10:44:34.639 observatory.py:342   Waiting for Dec tracking adjustment
D0115 10:44:39.290 observatory.py:342   Waiting for Dec tracking adjustment
D0115 10:44:43.951 observatory.py:342   Waiting for Dec tracking adjustment
D0115 10:44:48.610 observatory.py:342   Waiting for Dec tracking adjustment
D0115 10:44:53.260 observatory.py:342   Waiting for Dec tracking adjustment
W0115 10:44:57.921 tracking.py:15       Problem adjusting tracking: Trying to adjust Dec tracking for more than 30 seconds

```

**Comment by wtgee** [comment]
This happened multiple times last night after implementing the wait between RA and Dec axis (see #350). Each time it got stuck the correction it tried to do was less than 20ms, with the largest one that causes an error being 16.74ms. I see a 45ms correction that did work. Since this is about ~0.2 arcsec = ~1/50 pixel, I think we should just not try to correct unless it's larger than some certain amount.

Note that in #349 I want to speed up the rate, which would effect the amount of correction to do.

**Comment by wtgee** [comment]
Verified fix on PAN001 after #355 merge.
